helm-chart/renku/values.yaml

```diff
@@ -1498,6 +1498,32 @@ dataService:
   trustedProxies:
     proxiesCount: 1
     realIpHeader: x-real-ip
+  # Build session images from code repositories with Shipwright
+  imageBuilders:
+    # Experimental feature: enable at your own risk!
```

Comment:
I think we can add a bit more information here. At least that it requires shipwright to be installed independently from the renku helm chart.